app/workers/bgs/submit_form686c_job.rb

```diff
@@ -11,13 +11,23 @@ class Invalid686cClaim < StandardError; end
 
     def perform(user_uuid, saved_claim_id, vet_info)
       user = User.find(user_uuid)
+      claim = valid_claim(saved_claim_id, vet_info)
+
+      BGS::Form686c.new(user).submit(claim.parsed_form)
+    rescue
+      DependentsApplicationFailureMailer.build(user).deliver_now if user.present?
```

Comment:
Are we sure we want to `deliver_now`? Is there a performance cost? I don't even know if we have a mail queue, so it's possible this question is philosophical.

Reply:
@f1337 Ah yes, I am open to suggestions.

A search through the project did not return `#deliver_later`, just `#deliver_now`. Also, I figured since we're firin' this in a job, it shouldn't be too much of an issue.

So with the above in mind, I went with `#deliver_now`. This is my first mailer on this project so I could just be naive to what's available.